### ISSUE TITLE
Performance dropped largely when using batch size = 8

### ISSUE THREAD

**Issue by adaxidedakaonang** [zhangchbin/RCIL#12]
Hi, firstly thanks for offering the code. When I trained network with batch size =8, initial lr= 0.01 and incremental lr = 0.001, the performance dropped larger after third step under disjoint 15-1. I want to compare my method with yours, but the performance of 15-1 is only 16% mIoU. For the limitation of the GPU, I cant use a large batch size. Below is the detailed performance.
`0, 0.7819295688373131
1, 0.724357085122224
2, 0.6405337621460513
3, 0.572540403139553
4, 0.17674935488159058
5, 0.14200443360724102
`
Im very confused about this result, it' too low, but I didn't change the code.

**Comment by zhangchbin** [comment]
We use SGD optimizer, so if you change the batch size to 8, don't forget to modify the learning rate. If you have any results, please let me know.

**Comment by adaxidedakaonang** [comment]
I have changed the initial lr from 0.02 to 0.01. Anyway, I will try to train step 4th and step 5th with lr 0.0005. The reason for using batch size 8 is to keep parameters the same as other methods.

**Comment by zhangchbin** [comment]
batch_size_1 / lr_1 should be equal to batch_size_2 / lr_2. You can refer to the document of mmdetection. By the way, MiB and PLOP should use batch_size=24.

**Comment by adaxidedakaonang** [comment]
Thank you very much, I will try with 0.0003

**Comment by zhangchbin** [comment]
hi, @adaxidedakaonang 
We found that high version pytorch can not reach the supposed performance due to the bug caused by InplaceABNSync, which has been fixed in our latest commitment. You can refer to our latest news. Sorry for the inconvenience caused.
